finality-aleph/src/lib.rs

```diff
@@ -269,20 +270,16 @@ impl<H: Header<Number = BlockNumber>> BlockIdentifier for BlockId<H> {
     }
 }
 
-pub struct AlephConfig<B, H, C, SC, CS>
-where
-    B: Block,
-    B::Header: Header<Number = BlockNumber>,
-    H: ExHashT,
-{
-    pub network: Arc<NetworkService<B, H>>,
+pub struct AlephConfig<C, SC, CS> {
+    pub network: Arc<NetworkService<AlephBlock, AlephHash>>,
     pub client: Arc<C>,
     pub chain_status: CS,
+    pub import_queue_handle: Box<dyn ImportQueueService<AlephBlock>>,
```

Comment:
why Box<dyn> over generic?

Reply:
this is what we get from substrate

Reply:
Ugh, it would be nice to wrap it (as I think we are trying to avoid traits here?) at some point, because the type is quite ugly. This is not a blocker for this PR imo, but if you have an idea how to do this then please do.

Reply:
`AlephConfig` needs to disappear...

IDK, maybe we want a type alias in `primitives`, similar to `Block`, `Header` etc.? How should we call it then, `BlockImport`?

Reply:
no, w8, this makes little sense - `run_validator_node` should use trait bounds from submodules, as it's an interface to the whole crate, let's discuss this properly

Reply:
wrapped